README.md

```diff
@@ -32,30 +32,27 @@ Example of listing all the pods in `kube-system` namespace.
 Results are streamed, limited to 20 results per page.
 
 ```rust
-extern crate kubernetes;
-extern crate tokio_core;
-
 use std::default::Default;
-use tokio_core::reactor::Core;
+use hyper::rt;
 
-use kubernetes::api;
-use kubernetes::client::{Client,ListOptions};
-use kubernetes::api::core::v1::{Pod,PodList};
+use kubernetes_api::core::v1::Pods;
+use kubernetes_apimachinery::meta::NamespaceScope;
+use kubernetes_apimachinery::meta::v1::ListOptions;
+use kubernetes_client::{Client,ListOptions};
```

Comment:
I worry about grabbing so many top level names; are they all free?  kubernetes we already have right?

Reply:
Agreed.  I can roll everything into a single crate if we prefer. I think the only one I might want separate is the "api" crate, since this needs a release for each upstream k8s release and i think it will be easier to manage that on a separate lifecycle.

Reply:
If the names are free I have no objection to the sprawl. lets see ... seems ok. We can always consolidate later.